experiments/space_influenza_cv.ipynb

```diff
@@ -63,7 +63,7 @@
      "name": "stderr",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #15.            new_y = pmb.utils._sample_posterior(
</pre><p>It seems you forgot to transform the predictions. This probably explains why the rmsd look so bad</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Grupo-de-modelado-probabilista/BART/pull/51/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>